packages/gestalt/src/SearchField.js

```diff
@@ -81,82 +87,92 @@ const SearchFieldWithForwardRef: React$AbstractComponent<
   const hasValue = value && value.length > 0;
   const hideSearchIcon = focused || hasValue;
 
-  const className = classnames(styles.input, {
-    [layout.medium]: size === 'md',
-    [layout.large]: size === 'lg',
-    [styles.inputActive]: focused || hasValue,
-    [styles.inputHovered]: hovered,
-  });
+  const className = classnames(
+    styles.input,
+    {
+      [layout.medium]: size === 'md',
+      [layout.large]: size === 'lg',
+      [styles.inputActive]: focused || hasValue,
+      [styles.inputHovered]: hovered,
+    },
+    hasError || errorMessage ? formElement.errored : formElement.normal
+  );
 
   const clearButtonSize = size === 'lg' ? 24 : 20;
   const clearIconSize = size === 'lg' ? 12 : 10;
 
   return (
-    <Box
-      alignItems="center"
-      display="flex"
-      onMouseEnter={handleMouseEnter}
-      onMouseLeave={handleMouseLeave}
-      onFocus={handleFocus}
-      onBlur={handleBlur}
-      position="relative"
-    >
-      {!hideSearchIcon && (
-        <Box
-          dangerouslySetInlineStyle={{
-            __style: {
-              pointerEvents: 'none',
-              // Added the following lines for Safari support
-              top: '50%',
-              transform: 'translateY(-50%)',
-            },
-          }}
-          left
-          right
-          paddingX={4}
-          position="absolute"
-        >
-          <Icon icon="search" accessibilityLabel="" />
-        </Box>
-      )}
-      <input
-        ref={ref}
-        aria-label={accessibilityLabel}
-        autoComplete={autoComplete}
-        className={className}
-        id={id}
-        onChange={handleChange}
-        placeholder={placeholder}
-        role="searchbox"
-        type="search"
-        value={value}
-      />
-      {hasValue && (
-        <button
-          className={styles.clear}
-          onClick={handleClear}
-          tabIndex={-1}
-          type="button"
-        >
+    <span>
```

Comment:
Instead of wrapping this in a `<span />` - could we see it if works with a React Fragment?

Reply:
Changed this to React Fragment